tempesta_fw/t/functional/helpers/control.py

```diff
@@ -297,6 +371,7 @@ def __init__(self, stap_script, mod=False, mod_name='stap_tempesta'):
             self.modules_dir = '/lib/modules/$(uname -r)/custom/'
         else:
             self.stap_msg = 'Cannot %s stap %s kernel.'
+        self.stop_procedures = [self.stop_tempesta, self.remove_config]
```

Comment:
Already defined in base `Tempesta` class. no need to redefine here.

Reply:
for some reason, without whis explicit definition, overriding worked bad and original functions called